net-p2p/retroshare/retroshare-0.6.7.ebuild

```diff
@@ -23,6 +23,7 @@ REQUIRED_USE="
 	?? ( libupnp miniupnp )
 	service? ( || ( cli jsonapi ) )"
 
+# NOTE(setan): rapodjson might not be needed at runtime if not using jsonapi
```

Comment:
Being an header only library we probably doesn't need it at runtime in any case, I bet it is needed at compile time only (and most probably just if JSON API is enabled, but this last part should be tested before being sure)

Reply:
So you are saying that it could be moved to the CBUILD, so to  `DEPEND`?

Reply:
yes